test/src/test/java/hudson/slaves/SlaveComputerTest.java

```diff
@@ -156,7 +156,7 @@ public void startupShouldFailOnErrorOnlineListener() throws Exception {
     @TestExtension(value = "startupShouldFailOnErrorOnlineListener")
     public static final class ErrorOnOnlineListener extends ComputerListener {
 
-        static int onOnlineCount = 0;
+        volatile static int onOnlineCount = 0;
```

Comment:
Checkstyle enforcement wants something different?
```suggestion
        static volatile int onOnlineCount = 0;
```